invenio_app_rdm/records_ui/previewer/iiif_simple.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2022 data-futures.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Simple IIIF image preview."""
+
+from os.path import splitext
+
+from flask import current_app, render_template
+
+previewable_extensions = ['gif', 'jpg', 'jpeg', 'jp2', 'png', 'tif', 'tiff']
+
+# extensions that can be put in <img> tag - all others converted to jpg
+browser_extensions = ['gif', 'jpg', 'png']
+
+
+def can_preview(file):
+    """Check if file can be previewed by this previewer.
+
+    :param file: The file to be previewed.
+    :returns: Boolean
+    """
+    supported_extensions = ['.gif', '.jpg', '.jpeg', '.jp2', '.png',
```

Comment:
Minor: I guess this list is based on what Pillow supports, or what we know that works, at the present moment. My only concern is that if that changes in the future it requires a module update and release vs. just a config change (or just updating Pillow to the latest/desired version).

I see two solutions:

1. Introduce a configuration variable, e.g. `IIIF_SIMPLE_PREVIEWER_SUPPORTED_EXTENSIONS = [...]`
2. Use something like the following to auto-populate the list:

```python
from PIL import Image
Image.registered_extensions()
# returns the following:
{'.blp': 'BLP',
 '.bmp': 'BMP',
 '.dib': 'DIB',
 '.bufr': 'BUFR',
 '.cur': 'CUR',
 '.pcx': 'PCX',
 '.dcx': 'DCX',
 '.dds': 'DDS',
 '.ps': 'EPS',
 '.eps': 'EPS',
 '.fit': 'FITS',
 '.fits': 'FITS',
 '.fli': 'FLI',
 '.flc': 'FLI',
 '.ftc': 'FTEX',
 '.ftu': 'FTEX',
 '.gbr': 'GBR',
 '.gif': 'GIF',
 '.grib': 'GRIB',
 '.h5': 'HDF5',
 '.hdf': 'HDF5',
 '.png': 'PNG',
 '.apng': 'PNG',
 '.jp2': 'JPEG2000',
 '.j2k': 'JPEG2000',
 '.jpc': 'JPEG2000',
 '.jpf': 'JPEG2000',
 '.jpx': 'JPEG2000',
 '.j2c': 'JPEG2000',
 '.icns': 'ICNS',
 '.ico': 'ICO',
 '.im': 'IM',
 '.iim': 'IPTC',
 '.tif': 'TIFF',
 '.tiff': 'TIFF',
 '.jfif': 'JPEG',
 '.jpe': 'JPEG',
 '.jpg': 'JPEG',
 '.jpeg': 'JPEG',
 '.mpg': 'MPEG',
 '.mpeg': 'MPEG',
 '.mpo': 'MPO',
 '.msp': 'MSP',
 '.palm': 'PALM',
 '.pcd': 'PCD',
 '.pdf': 'PDF',
 '.pxr': 'PIXAR',
 '.pbm': 'PPM',
 '.pgm': 'PPM',
 '.ppm': 'PPM',
 '.pnm': 'PPM',
 '.psd': 'PSD',
 '.bw': 'SGI',
 '.rgb': 'SGI',
 '.rgba': 'SGI',
 '.sgi': 'SGI',
 '.ras': 'SUN',
 '.tga': 'TGA',
 '.icb': 'TGA',
 '.vda': 'TGA',
 '.vst': 'TGA',
 '.webp': 'WEBP',
 '.wmf': 'WMF',
 '.emf': 'WMF',
 '.xbm': 'XBM',
 '.xpm': 'XPM'}
```

I have a slight preference for `1.` (i.e. the config variable) since it's technically a whitelist that we can control easily in case e.g. there's a security vulnerability found for a type of extension. Still it allows someone brave to just set `IIIF_SIMPLE_PREVIEWER_SUPPORTED_EXTENSIONS = Image.registered_extensions().keys()` :)

Reply:
Done